test/activation.spec.ts

```diff
@@ -13,23 +13,26 @@ import {
 } from '../src';
 import { ValueOf, createPipelineState, MockPipelineState, MockInstruction } from './shared';
 
-describe('activation', () => {
-  describe('CanDeactivatePreviousStep', () => {
+describe('activation', function activation() {
```

Comment:
just curious why did you name this function?

Reply:
In vscode outline view, we have nice view with named function:
![image](https://user-images.githubusercontent.com/9994529/47325338-fa7dc480-d6ae-11e8-8117-443f12d4cb61.png)
(Each nested fn is a `describe`)
It aids navigation when the test file becomes too lengthy. I think we should do this for vNext too.

Reply:
ooh really now! I had no idea. That's awesome